client/components/Players/PlayerManager.vue

```diff
@@ -295,6 +340,125 @@ export default Vue.extend({
             window.addEventListener('keydown', this.handleKeyPress);
           }
 
+          break;
+        case 'playbackManager/INCREASE_QUEUE_INDEX':
+        case 'playbackManager/DECREASE_QUEUE_INDEX':
+        case 'playbackManager/SET_CURRENT_ITEM_INDEX':
+          this.upNextUserHidden = false;
```

Comment:
No need to subscribe for this, you can simply watch the current item index. If it changes, reset this to false.